CMakeLists.txt

```diff
@@ -34,6 +34,18 @@ option(RAY_USE_NEW_GCS
   "Use the new GCS implementation"
   OFF)
 
+option(RAY_USE_GLOG
+  "Build the logging system using glog"
+  ON)
+
+option(RAY_USE_LOG4CPLUS
```

Comment:
What is the benefit of having both log4cplus and glog support? I prefer to standardize on one (glog) and support it really well. If we support both, the maintenance overhead is increased.